src/UI/Implementation/Component/Listing/DescriptiveList/DescriptiveList.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\Listing\DescriptiveList;
+
+use ILIAS\UI\Component as C;
+
+/**
+ * Class DescriptiveList
+ * @package ILIAS\UI\Implementation\Component\Listing\DescriptiveList
+ */
+class DescriptiveList implements C\Listing\DescriptiveList {
+
+	/**
+	 * @var	array
+	 */
+	private  $items;
+
+	/**
+	 * @inheritdoc
+	 */
+	public function __construct(array $items) {
```

Comment:
Please check given items, as your renderer will explode when somebody passes something you do not expect. Better make invalid state unrepresentable by denying wrong inputs up front.